README.md

```diff
@@ -46,6 +49,10 @@ jobs:
           # required
           app_id: ${{ vars.APP_ID }}
           private_key: ${{ secrets.PRIVATE_KEY }}
+          # optional - if not used, defaults to current repository owner
+          owner: ${{ github.repository_owner }}
+          # optional - if not used, defaults to all repositories in the organization where the app is installed
+          repositories: repo1,repo2
```

Comment:
remove from this usage example, I'd focus on the use case

Reply:
Done.